code-challenges.test.js

```diff
@@ -38,36 +95,75 @@ const studyMinutesWeek1 = {
   friday: 15,
   saturday: 60
 }
-// Expected output: [15, 15, 20, 30, 30, 60, 90]
-
-const studyMinutesWeek2 = {
-  sunday: 100,
-  monday: 10,
-  tuesday: 45,
-  wednesday: 60,
-  thursday: 20,
-  friday: 15,
-  saturday: 65
+// // //Expected output: [15, 15, 20, 30, 30, 60, 90]
+
+// // const studyMinutesWeek2 = {
+// //   sunday: 100,
+// //   monday: 10,
+// //   tuesday: 45,
+// //   wednesday: 60,
+// //   thursday: 20,
+// //   friday: 15,
+// //   saturday: 65
+// // }
+// // Expected output: [10, 15, 20, 45, 60, 65, 100]
+
+
+// // b) Create the function that makes the test pass.
+//Create a function and declare a variable that takes in an object
+
+const sortedLeast = (object) => {
+  const studyMinutesWeek= (studyMinutesWeek1.value)
+  studyMinutesWeek1.sort((a,b)=> a-b)
+  return sortedLeast
 }
-// Expected output: [10, 15, 20, 45, 60, 65, 100]
+// The value is required so we create a variable to store the objects value
+//usingh .sort method, we are able to sort the arrValue numbers from least to greatest.
 
-// b) Create the function that makes the test pass.
+//ERRORS:
+//TypeError: Cannot read properties of undefined (reading 'value')
```

Comment:
This error returned because `.value` is not defined anywhere. I think you just got confused when reading the `Object.values` resource. To access one value in an object, we use dot notation on the object followed by the name of the key, which will return the value at that key. So for example, we can do `studyMinutesWeek2.friday` and it will return 15. The resource given in the assessment talks about `Object.values()`, which is used differently. To use it, we need to literally code `Object.values()`, and then pass in the object into the parentheses. This will return an array of all of the values in that object. So for example, we can do `Object.values(studyMinutesWeek2)` and that will return `[100, 10, 45, 60, 20, 15, 65]`. Then from there we can use `.sort` on that array.